BrowserKit/Package.swift

```diff
@@ -6,7 +6,7 @@ let package = Package(
     name: "BrowserKit",
     platforms: [
         .iOS(.v15),
-        .macOS(.v10_15)
+        .macOS(.v12)
```

Comment:
I think this is ok since we don't really build for mac ? but the lib needs macos 12 or later

Reply:
Yes exactly because of markdownkit. They set a minimum of macos12 in [their spm file](https://github.com/bmoliveira/MarkdownKit/blob/master/Package.swift#L9-L10).